src/main/java/com/arpnetworking/metrics/mad/parsers/StatsdToRecordParser.java

```diff
@@ -154,16 +163,19 @@ private Number parseValue(
     @SuppressFBWarnings("NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE")
     // See: https://github.com/findbugsproject/findbugs/issues/79
     private ImmutableMap<String, String> parseTags(@Nullable final String taqsAsString) {
-        final ImmutableMap.Builder<String, String> annotations = ImmutableMap.builder();
         if (null != taqsAsString) {
-            for (final String keyValue : taqsAsString.split(",")) {
-                final int pivot = keyValue.indexOf(':');
-                annotations.put(
-                        keyValue.substring(0, pivot),
-                        keyValue.substring(pivot + 1));
-            }
+            return ImmutableMap.copyOf(Splitter.on(',').withKeyValueSeparator(':').split(taqsAsString));
+        }
+        return ImmutableMap.of();
+    }
+
+    @SuppressFBWarnings("NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE")
+    // See: https://github.com/findbugsproject/findbugs/issues/79
+    private ImmutableMap<String, String> parseInfluxStyleTags(@Nullable final String taqsAsString) {
```

Comment:
Nit: typo in argument name, should be tagsAsStrings not taqsAsStrings, same with original function.

Reply:
Fixed, thanks.